requirements26.txt

```diff
@@ -1,2 +1,2 @@
 pycparser==2.18
-paramiko==2.3.2
+paramiko==2.3.3
```

Comment:
patch is not complete, see https://github.com/awslabs/cfncluster-node/blob/7c88f96a5633328a0e3c0b9f71eee377cd63942a/setup.py#L35